File: 준영/LV2/기능개발.py
def solution(progresses, speeds):
    answer = []
    n = len(progresses)
    arr = [0] * n
    for i in range(n):
        if (100 - progresses[i]) % speeds[i] == 0:
            arr[i] = (100 - progresses[i]) // speeds[i]
        else:
            arr[i] = (100 - progresses[i]) // speeds[i] + 1
    temp = arr[0]
    result = 1
    for i in range(1, n):
        if temp < arr[i]:
            answer.append(result)
            result = 1
            temp = arr[i]
        else:
            result += 1
    answer.append(result)
    return answer

File: 준영/프로그래머스/LV2/기능개발.js
function solution(progresses, speeds) {
  var answer = []
  var n = progresses.length
  var arr = Array.from({length: n}, x => 0)
  for (let i = 0; i < n; i++) {
    if ((100 - progresses[i]) % speeds[i] === 0) {
      arr[i] = parseInt((100 - progresses[i])/speeds[i])
    } else {
      arr[i] = parseInt((100 - progresses[i])/speeds[i]) + 1
    }
  }
  var temp = arr[0]
  var result = 1
  for (let i = 1; i < n; i++) {
    if (temp < arr[i]) {
      answer.push(result)
      result = 1
      temp = arr[i]
    } else {
      result += 1
    }
  }
  answer.push(result)

  return answer
}